android/app/build.gradle

```diff
@@ -172,6 +171,12 @@ dependencies {
     compile 'com.pusher:pusher-websocket-android:0.5.0'
 }
 
+subprojects {
+    if(name.equals('react-native-google-analytics-bridge')) {
```

Comment:
Yes yes yes. I think this should take care of the bug I always experience in Android Studio

Reply:
Haha yeah... Not sure why I wasn't getting it until now